docs/specification/draft/server/tools.mdx

```diff
@@ -264,6 +267,7 @@ Example tool with output schema:
 ```json
 {
   "name": "get_weather_data",
+  "displayName": "Weather Data Retriever",
```

Comment:
This doesn't appear to be represented in the spec change (and is duplicative with existing `ToolAnnotations.title`